lib/cli/interactive-setup/index.js

```diff
@@ -64,6 +64,11 @@ module.exports = async (context) => {
     } else {
       context.isConsole = true;
     }
+  } else if (
+    (!options.console && !options.dev && initialContext.isInServiceContext) ||
+    options.app
+  ) {
+    context.isDashboard = true;
   }
```

Comment:
@medikoo do you have a suggestion on how to set our `isDashboard` flag? This appears to cause issues with the test never completing and I can't tell which test is the one that is stalling 🤔

Reply:
I believe it should be set to `!options.console && !options.dev` , and this setting should be unconditional (not in any deep logic that's specific to `isConsole` etc resolution)

Reply:
Ahh okay got it. Thanks man!